### PR TITLE
Ignore option

### PR DESCRIPTION
**Pull request by yorinasub17**
For @woohp 

Add option to ignore package managers from installing.

### CODE REVIEW

**Comment by yorinasub17** [comment]
Don't merge until I make the corresponding "install only" option

**Comment by yorinasub17** [comment]
@woohp Ok this is now ready, as I have added the corresponding `--install-only` option

File: scripts/cappa [hunk 1]
```diff
@@ -19,13 +19,27 @@ def cappa():
               help='Use oauth authenticated https for private repositories (requires GITHUB_TOKEN env var)')
 @click.option('--use-venv/--no-venv', default=True,
               help='Assumes install uses a python virtualenv')
+@click.option('--ignore-managers', default='',
```
Comment (woohp) [inline scripts/cappa]: can we use `--ignore` and `--only`?

**Comment by yorinasub17** [comment]
Changed. Merging now!

**Pull request merged**